CONTRIBUTING.md

```diff
@@ -8,54 +8,86 @@ $ npm install
 $ npm run bootstrap
 ```
 
-Then you can either run:
+#### Build
 
 ```sh
-$ npm run build
+npm run build
 ```
 
-to build Babel **once** or:
+to build **once**
 
 ```sh
-$ npm run watch
+npm run watch
 ```
 
-to have Babel build itself then incrementally build files on change.
+to do incremental builds in watch mode
 
-To run all tests:
+#### Lint
+
+The project uses [prettier](https://github.com/prettier/prettier) for formatting code and [eslint](https://github.com/eslint/eslint) for other linting.
```

Comment:
Similar to above, maybe:

This project uses [prettier](https://github.com/prettier/prettier) for formatting code and [eslint](https://github.com/eslint/eslint) for other linting.

To check both:

```sh
npm run lint
```

You can also run them each individually:

 ```sh
# prettier
npm run format-check

# eslint
npm run eslint
```